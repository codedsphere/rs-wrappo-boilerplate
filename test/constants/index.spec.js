import * as constants from 'mt-constants';

describe('constants', () => {
  it('should have the correct constants', () => {
    expect(constants).toMatchSnapshot();
  });
});
