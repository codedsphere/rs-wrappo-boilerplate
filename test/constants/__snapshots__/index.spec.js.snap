// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should have the correct constants 1`] = `
Object {
  "API": Object {
    "ADD_BOOKMARKS": "/api/v1/account/webapi/account/profile/addBookmarks",
    "CALIBRATE": "/api/v1/account/webapi/account/users/calibrate",
    "CONTENTS": "/api/v1/cms/internal/pc/31/contents",
    "CURRENT_RESULTS": "/api/v1/result/service/mobile/results/current",
    "DEL_BOOKMARKS": "/api/v1/account/webapi/account/profile/delBookmarks",
    "GET_BOOKMARKS": "/api/v1/account/webapi/account/profile/getBookmarks",
    "GET_VALIDATE_PIC": "/api/v1/account/webapi/account/validateCode/getValidatePic?webUniqueCode=",
    "HbInfos": "/api/v1/balance/me/hb/current/31",
    "LAST_AND_CURRENT_RESULT": "/api/v1/result/service/mobile/results/currentTwo",
    "PRE_REGISTER_GUEST": "/api/v1/account/webapi/account/users/preRegisterGuest",
    "PRIZE_SETTING": "/api/v1/adminsettings/user/prizeSettings?clientId=31",
    "REGISTER_NEW_GUEST": "/api/v1/account/webapi/account/users/registerNewGuest",
    "REGISTER_NEW_GUEST2": "/api/v1/account/webapi/account/users/registerNewGuest2",
    "REGISTER_NEW_USER": "/api/v1/account/webapi/account/users/registerNewUser",
    "REGISTER_NEW_USER2": "/api/v1/account/webapi/account/users/registerNewUser2",
    "REGISTRY_SWITCHS": "/api/v1/account/webapi/operate/users/registrySwitchs",
    "START_CAPTCHA": "/api/v1/account/webapi/account/userss/startCaptcha",
    "UNIQUE_GAME_RESULTS": "/api/v1/result/service/mobile/results/hist",
    "WEB_LOGIN": "/api/v1/account/webapi/account/users/encryptLogin",
    "affCode": "/api/v1/account/webapi/team/affiliates",
    "affCodeList": "/api/v1/account/webapi/team/affiliates/list",
    "affCodeUrl": "/api/v1/account/webapi/team/affiliates/affCode",
    "allHistory": "/api/v1/result/service/mobile/results/lastOpen",
    "allResults": "/api/v1/result/service/mobile/results/lastOpen",
    "annoucementsList": "/api/v1/cms/internal/announcement?adminId=31",
    "balanceHistory": "/api/v1/balance/me/history",
    "bankList": "/api/v1/cashmgt/me/transfer/topups/banktransfers/banklist/v2/?id=31",
    "bankTransfers": "/api/v1/cashmgt/me/transfer/topups/banktransfers/v3",
    "banktransfersQuery": "/api/v1/cashmgt/me/transfer/topups/banktransfers/v3",
    "cancelOrder": "/api/v1/ordercap/me/cancel",
    "changePwd": "/api/v1/account/webapi/account/users/change/encryptPassword",
    "checkIpInfo": "/api/v1/update/checkIpInfo",
    "clientID": 31,
    "commissionDetail": "/api/v1/balance/me/commissions/details",
    "createDownline": "/api/v1/account/webapi/account/users/createEncryptUser",
    "currentResultsOnly": "/api/v1/result/service/mobile/results/currentOnly",
    "downloadLink": "/config/downConfig.json",
    "extraRegisterInfos": "/api/v1/account/webapi/operate/users/registrySwitchs",
    "findTopWinners": "/api/v1/orderdata/me/orders/findTopWinners?clientId=31",
    "gameList": "/api/v1/gameinterface/user/showRNGGame",
    "gameLoginUrl": "/api/v1/gameinterface/user/loginurl",
    "getGt": "/api/v1/account/webapi/account/userss/startCaptcha",
    "getHistoryList": "/api/v1/result/service/mobile/results/today",
    "getValidatePic": "/api/v1/account/webapi/account/validateCode/getValidatePic",
    "helpList": "/api/v1/cms/internal/helpList/31",
    "homeInfo": "/api/v1/cms/internal/pc/31/contents",
    "login": "/api/v1/account/webapi/account/users/login",
    "loginHistory": "/api/v1/account/webapi/account/users/loginHistory",
    "logout": "/api/v1/account/account/system/logout",
    "memberList": "/api/v1/account/webapi/team/users/list",
    "messageList": "/api/v1/cms/Message/playerMessages",
    "myCashFlow": "/api/v1/balance/me/details",
    "myCommissions": "/api/v1/balance/me/commissions/",
    "orderDetail": "/api/v1/orderdata/me/orders/findByTimeuuid",
    "orderHistory": "/api/v1/orderdata/me/orders/findByState",
    "ordercap": "/api/v1/ordercap/me/order",
    "paymentList": "/api/v1/cashmgt/me/transfer/topups/payment/list",
    "personalReport": "/api/v1/balance/me/statements/user",
    "playerBalanceAllBalanceTransferToCenter": "/api/v1/dsf/center/player/balance/allBalanceTransferToCenter",
    "playerBalanceTransfer": "/api/v1/dsf/center/player/balance/transfer",
    "playerGames": "/api/v1/dsf/center/player/31/games",
    "playerGetBalance": "/api/v1/dsf/center/player/getBalance",
    "playerGetBetLog": "/api/v1/dsf/center/player/getBetLog",
    "playerGetTransfer": "/api/v1/dsf/center/player/getTransfer",
    "playerOpenPlatform": "/api/v1/dsf/center/player/open/platform?clientId=31",
    "playerPlatforms": "/api/v1/dsf/center/player/platforms",
    "playerRegister": "/api/v1/dsf/center/player/register",
    "playerStartDemoGame": "/api/v1/dsf/center/player/startDemoGame",
    "playerStartGame": "/api/v1/dsf/center/player/startGame",
    "playerStatementsPersonalList": "/api/v1/dsf/center/player/statements/personal/list",
    "playerStatementsPersonalTotal": "/api/v1/dsf/center/player/statements/personal/total",
    "playerTeamCalc": "/api/v1/dsf/center/player/teamCalc",
    "playerTeamCalcMy": "/api/v1/dsf/center/player/teamCalc/my",
    "playerTeamCalcMyDay": "/api/v1/dsf/center/player/teamCalc/my/day",
    "playerTeamCalcMyList": "/api/v1/dsf/center/player/teamCalc/my/list",
    "preRegisterGuest": "/api/v1/account/webapi/account/users/preRegisterGuest",
    "providerBalance": "/api/v1/gameinterface/user/walletprovider",
    "refreshToken": "/api/v1/account/account/system/refreshToken/",
    "register": "/api/v1/account/webapi/account/users/register",
    "registerGuest": "/api/v1/account/webapi/account/users/registerGuest",
    "reloadProviderBalance": "/api/v1/gameinterface/user/wallettransfer",
    "specialOfferList": "/api/v1/cms/internal/promotion?contentType=PC_PROMOTION&adminId=31",
    "teamReport": "/api/v1/balance/me/statements/team",
    "teamReportQuery": "/api/v1/balance/me/statements/team/query",
    "teamReportSummary": "/api/v1/balance/me/statements/team/summary",
    "topupGroups": "/api/v1/cashmgt/me/payments/name",
    "topups": "/api/v1/cashmgt/me/transfer/topups",
    "transactionHistory": "/api/v1/cashmgt/me/transfer/orderhistory",
    "trendResult": "/api/v1/result/service/mobile/results/hist/",
    "uniqueGameHistory": "/api/v1/result/service/mobile/results/hist",
    "updateBankInfo": "/api/v1/account/webapi/account/users/encryptRegister_info",
    "updateRealName": "/api/v1/account/webapi/account/users/updateRealName",
    "updateUserInfo": "/api/v1/account/webapi/account/users",
    "updateVersion": "/api/v1/cms/internal/lastVersion",
    "userBalance": "/api/v1/balance/me/balance/details",
    "userBankOptions": "/api/v1/cashmgt/me/cards/list",
    "userBanksAccount": "/api/v1/cashmgt/me/cards/cardsAndWithdrawDetail",
    "userCards": "/api/v1/cashmgt/me/cards",
    "userCheckIn": "/api/v1/account/webapi/operate/users/signIn",
    "userId": "/api/v1/account/webapi/account/users/chekcUserId",
    "userInfo": "/api/v1/account/webapi/account/users/current",
    "userWithDraw": "/api/v1/cashmgt/me/transfer/withdrawalsV2",
    "validatePic": "/api/v1/account/webapi/account/validateCode/validatePic",
    "webRegister": "/api/v1/account/webapi/account/users/userWebEncryptRegister",
    "webRegisterGuest": "/api/v1/account/webapi/account/users/webEncryptRegisterGuest",
  },
  "Reducer": Object {
    "appModel": Object {
      "INITIALIZE_ALL": "appModel/INITIALIZE_ALL",
      "INITIALIZE_STATE": "appModel/INITIALIZE_STATE",
      "REMOVE_STATE": "appModel/REMOVE_STATE",
      "UPDATE_STATE": "appModel/UPDATE_STATE",
      "namespace": "appModel",
    },
    "pageModel": Object {
      "INITIALIZE_ALL": "pageModel/INITIALIZE_ALL",
      "INITIALIZE_STATE": "pageModel/INITIALIZE_STATE",
      "REMOVE_STATE": "pageModel/REMOVE_STATE",
      "UPDATE_STATE": "pageModel/UPDATE_STATE",
      "namespace": "pageModel",
    },
    "themeModel": Object {
      "INITIALIZE_ALL": "themeModel/INITIALIZE_ALL",
      "INITIALIZE_STATE": "themeModel/INITIALIZE_STATE",
      "Other_Reducer": "themeModel/Other_Reducer",
      "REMOVE_STATE": "themeModel/REMOVE_STATE",
      "UPDATE_STATE": "themeModel/UPDATE_STATE",
      "namespace": "themeModel",
    },
    "userModel": Object {
      "INITIALIZE_ALL": "userModel/INITIALIZE_ALL",
      "INITIALIZE_STATE": "userModel/INITIALIZE_STATE",
      "REMOVE_STATE": "userModel/REMOVE_STATE",
      "UPDATE_STATE": "userModel/UPDATE_STATE",
      "namespace": "userModel",
    },
  },
  "Saga": Object {
    "errorSaga": Object {
      "postErrorNotification": "errorSaga/postErrorNotification",
    },
    "pageSaga": Object {
      "callGitRepo": "pageSaga/callGitRepo",
      "getUserLogout": "pageSaga/getUserLogout",
      "putUserLogin": "pageSaga/putUserLogin",
    },
  },
  "SiteMap": Object {
    "Main": Object {
      "urls": Object {
        "/": Object {
          "placeholder": "Home",
        },
      },
    },
  },
}
`;
